site/docs/resources/faq.md

```diff
@@ -13,45 +13,45 @@ It reached the first stable version a few weeks later.
 
 ## How is grammY developed?
 
-grammY is completely free and open-source software, developed by a team of volunteers.
+grammY is a completely free and open-source software, developed by a team of volunteers.
 Its code is available on GitHub.
 
 You're welcome to [join us](https://t.me/grammyjs)!
 
 ## What programming language does grammY use?
 
-grammY is written from the ground up in TypeScript, a superset of JavaScript.
-It can therefore run on Node.js.
+grammY is written from the ground up in TypeScrip — a superset of JavaScript.
+Therefore, it runs on Node.js.
```

Comment:
```suggestion
grammY is written from the ground up in TypeScript, a superset of JavaScript.
It can therefore run on Node.js.
```

Reply:
I think the "—" means "which is", and if you remove it, you should say ", which is ...".